generators/languages/index.js

```diff
@@ -13,9 +13,39 @@ var LanguagesGenerator = generators.Base.extend({});
 
 util.inherits(LanguagesGenerator, scriptBase);
 
+var configOptions = {};
+
 module.exports = LanguagesGenerator.extend({
     constructor: function() {
         generators.Base.apply(this, arguments);
+
+        configOptions = this.options.configOptions || {};
+
+        // This makes it possible to pass `languages` by argument
+        this.argument('languagesArgument', {
```

Comment:
I had some problems setting it in the promt for some reason when it was not passed as an argument.